superset-frontend/src/views/CRUD/dashboard/DashboardList.tsx

```diff
@@ -21,20 +21,28 @@ import { t } from '@superset-ui/translation';
 import PropTypes from 'prop-types';
 import React from 'react';
 import rison from 'rison';
+import { Label } from 'react-bootstrap';
```

Comment:
Could this be the Label from `src/components/Label`?

Reply:
Did that PR already get merged? I'll rebase and use that.